src/utils/codecs.ts

```diff
@@ -0,0 +1,33 @@
+import { contexts as ldContexts, documentLoaderFactory } from '@transmute/jsonld-document-loader';
+// TODO: The DCC fork of vpq is not working at the moment
+// import * as vpqr from '@digitalcredentials/vpqr';
+import * as vpqr from '@digitalbazaar/vpqr';
+
+const didContext = require('@digitalcredentials/did-context');
+const ed25519 = require('ed25519-signature-2020-context');
+const DccContextV1Url = "https://w3id.org/dcc/v1";
+import { DccContextV1 } from "./contexts";
```

Comment:
Any reason to import from a local copy, instead of from the NPM module? (see https://github.com/digitalcredentials/dcc#usage for example).

Reply:
Interesting - I hadn't seen this usage of `dcc-context`. I was using [this sample code](https://github.com/digitalcredentials/sign-and-verify-core/blob/master/src/common.ts#L3) from `sign-and-verify-core` as a reference point. However, when I try to install `dcc-context` in the manner suggested here, I came across this error despite having a later version of Node.js than the error suggests:

![Screen Shot 2021-10-31 at 7 53 34 PM](https://user-images.githubusercontent.com/8096163/139605663-adb298d1-6835-48fe-8ddd-ad2ba1c069cf.png)

Reply:
@kezike oh weird.. 
`dcc-context` module does not have an "engine" directive in its package.json; I wonder where that error is coming from!

Let's debug -- what npm and Node version are you using? Do you have any further details on that error (like in the npm error log)?

Reply:
The wallet uses the dcc-context package -- you can see the usage here: https://github.com/digitalcredentials/learner-credential-wallet/blob/main/app/lib/documentLoader.ts#L3
(And it does not result in any npm version error).

Reply:
Never mind - I was able to resolve this issue after a system restart (computer's been a bit wonky lately)!